server/services/educationAndWorkPlanService.test.ts

```diff
@@ -290,4 +292,32 @@ describe('educationAndWorkPlanService', () => {
       expect(actual).toEqual(Error('Service Unavailable'))
     })
   })
+
+  describe('getEducation', () => {
+    const prisonNumber = 'A1234BC'
+    const userToken = 'a-user-token'
+
+    it('should get prisoner education', async () => {
+      // Given
```

Comment:
Does this test need to setup some mocks and expectations? 🤔 

Specifically I think you'll want to setup the mock `educationAndWorkPlanClient` to return an `EducationResponse` (from your test data builder `aValidEducationResponse` 👍 )

The implementation then takes that `EducationResponse` and calls the mapper with it, in order to return the `EducationDto` , so you'll want to setup an expected `EducationDto` (with a test data builder that you've yet to write 👍 😁 ), and then in the `// Then` block assert that thats what you get back